hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/compactions/OffPeakHours.java

```diff
@@ -87,6 +87,7 @@ private static boolean isValidHour(int hour) {
     OffPeakHoursImpl(int startHour, int endHour) {
       this.startHour = startHour;
       this.endHour = endHour;
+      LOG.info(this.toString());
```

Comment:
The log is not good, do not have info about the region.

Reply:
> The log is not good, do not have info about the region.

delete this log is better?

Reply:
> The log is not good, do not have info about the region.

I already delete the log in new pull request.